### PR TITLE
Load test mehrdad

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: load_to_rds/email_subscribers.py [hunk 1]
```diff
@@ -40,22 +40,23 @@ def send_genre_emails(scraped_data: list[dict]):
...
-                             "BccAddresses": [],
-                             "CcAddresses": []},
-                Message={
-                    "Subject": {
-                        "Data": f"New {format_genre_text(genre)} games for you!"
-                    },
-                    "Body": {
-                        "Html": {
-                            "Data": to_send
+            if len(genre_games) > 0:
```
Comment (Leon-JS02) [inline load_to_rds/email_subscribers.py:43]: Nice, no more empty emails.

File: load_to_rds/test_load_to_rds.py [hunk 2]
```diff
@@ -58,3 +59,31 @@ def test_transform_to_tuples():
                                          datetime(2020, 9, 18),
                                          True,
                                          "example.com")
+
+
+class TestCreateHtmlMessage:
```
Comment (Leon-JS02) [inline load_to_rds/test_load_to_rds.py:64]: Nice OOP test class.

**Review by Leon-JS02 — APPROVED** [review]
This should fix the email problem.

**Review by MichaelHassan — APPROVED** [review]
Emails fix nowed

**Pull request merged**